meson.build

```diff
@@ -252,7 +252,7 @@ libm = cc.find_library('m', required: false)
 libgcab = dependency('libgcab-1.0', version : '>= 1.0', fallback : ['gcab', 'gcab_dep'])
 gcab = find_program('gcab', required : get_option('tests'))
 bashcomp = dependency('bash-completion', required: false)
-python3 = find_program('python3')
+python3 = find_program('python3.8', 'python3', 'python3.9')
```

Comment:
Any reason it needs to be 3.8 first then 3?  I would think it should be 3 first.

Reply:
Yes it's important. In the FreeBSD 13.0 python3 is related to the 3.7 version and there is no py37-setuptool module in the pkg  (It conflicts with py38-setuptool when it's built from the source). So we have to check first if there is python3.8 to provide required dependencies.

Reply:
Yes it's important. In the FreeBSD 13.0 python3 is related to the 3.7 version and there is no py37-setuptool module in the pkg  (It conflicts with py38-setuptool when it's built from the source). So we have to check first if there is python3.8 to provide required dependencies.